cpp/oneapi/dal/algo/rbf_kernel/backend/cpu/compute_kernel_dense.cpp

```diff
@@ -31,6 +31,7 @@ using result_t = compute_result<task::compute>;
 using descriptor_t = detail::descriptor_base<task::compute>;
```

Comment:
You need to add implementation for dpcpp interfaces. but throw exception for gpu implementation